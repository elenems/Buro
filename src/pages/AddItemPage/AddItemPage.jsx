import React from "react";
import Container from "@material-ui/core/Container";
import Grid from "@material-ui/core/Grid";
import Header from "../../blocks/Header";
import AddForm from "../../components/AddForm";

export default function AddItemPage() {
  return (
    <Container fixed>
      <Grid justify='center' container spacing={6}>
        <Grid item>
          <AddForm />
        </Grid>
      </Grid>
    </Container>
  );
}
