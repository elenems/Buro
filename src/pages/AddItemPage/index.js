import AddItemPage from "./AddItemPage.jsx";
export default AddItemPage;
