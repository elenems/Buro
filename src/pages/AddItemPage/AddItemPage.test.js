import React from "react";
import { shallow } from "enzyme";
import AddItemPage from "../AddItemPage";

describe("Add Item Page", () => {
  it("Matches snapshot", () => {
    const wrapper = shallow(<AddItemPage />).debug();
    expect(wrapper).toMatchSnapshot();
  });
});
