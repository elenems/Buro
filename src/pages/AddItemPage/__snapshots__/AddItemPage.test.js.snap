// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Add Item Page Matches snapshot 1`] = `
"<WithStyles(ForwardRef(Container)) fixed={true}>
  <WithStyles(ForwardRef(Grid)) justify=\\"center\\" container={true} spacing={6}>
    <WithStyles(ForwardRef(Grid)) item={true}>
      <AddForm />
    </WithStyles(ForwardRef(Grid))>
  </WithStyles(ForwardRef(Grid))>
</WithStyles(ForwardRef(Container))>"
`;
